examples/sri-1/components/workgroup/invitation/invitation.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Agreement } from "../storage/agreement";
+import { BpiSubject } from "../identity/bpiSubject";
+export interface IInvitation {
+    id: string;
+    name: string;
+    sender: BpiSubject;
+    recipient: string;
+    workgroupId: string;
+    agreement: Agreement;
```

Comment:
Why is there a need to store agreement object as part of the invitation? When the recipient responds, he\she can see the agreement state based on the workgroup id?

Reply:
and agrement state can change between the time the invitation has been sent and the time the recipient reacts to it

Reply:
agreement state removed.